packages/babel-helper-create-class-features-plugin/src/features.ts

```diff
@@ -62,8 +66,8 @@ export function enableFeature(file, feature, loose) {
     }
   }
 
-  let resolvedLoose: void | true | false;
-  let higherPriorityPluginName: void | string;
+  let resolvedLoose: boolean;
+  let higherPriorityPluginName: string;
```

Comment:
These can also be `undefined`.

Reply:
would be nice to enable strict mode, while adding improving type coverage… - or at least `strictNullChecks` which will catch things like this

however, not sure if it can be easily done for single package… probably this would require to revert project references configuration and fix circular dependencies…